bakery/src/scripts/fetch_book_group.sh

```diff
@@ -2,7 +2,7 @@
 exec > >(tee "${COMMON_LOG_DIR}"/log >&2) 2>&1
 
 reference=$(cat "${BOOK_INPUT}"/version)
-[[ "$reference" = latest ]] && reference=master
+[[ "$reference" = latest ]] && reference=main
```

Comment:
Based on this change it looks like we'll now adopt the convention / assumption that the default branch will be `main` for content repos, is that correct? If so, should / will we update `ce-git-storage-spike` accordingly?

Reply:
Based on Slack convos, it sounds like we're going with `main` as the default branch for all content repos.

Reply:
[Resource ](https://stevenmortimer.com/5-steps-to-change-github-default-branch-from-master-to-main/)I used to update `ce-git-storage-spike`:


```
ce-git-storage-spike[master] $ git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
ce-git-storage-spike[master] $ git branch -m master main
ce-git-storage-spike[main] $ git push -u origin main
Total 0 (delta 0), reused 0 (delta 0)
remote: 
remote: Create a pull request for 'main' on GitHub by visiting:
remote:      https://github.com/openstax/ce-git-storage-spike/pull/new/main
remote: 
To github.com:openstax/ce-git-storage-spike.git
 * [new branch]      main -> main
Branch 'main' set up to track remote branch 'main' from 'origin'.
ce-git-storage-spike[main] $ git symbolic-ref refs/remotes/origin/HEAD refs/remotes/origin/main
ce-git-storage-spike[main] $ git push origin --delete master
To github.com:openstax/ce-git-storage-spike.git
 - [deleted]         master
ce-git-storage-spike[main] $ git status
On branch main
Your branch is up to date with 'origin/main'.

nothing to commit, working tree clean
```

with: https://docs.github.com/en/github/administering-a-repository/changing-the-default-branch